src/models/two_nodes.js

```diff
@@ -0,0 +1,886 @@
+import { round } from "../utilities/utilities.js";
+import { p_sat_torr } from "../psychrometrics/p_sat_torr.js";
+
+/**
+ * @typedef {Object} TwoNodesReturnType
```

Comment:
for the props in this type please follow the same naming conventions as the python library:
so `eSkin` -> `e_skin` etc